js/skin.js

```diff
@@ -287,8 +287,42 @@ var Skin = createReactClass({
     }
   },
 
+  /**
+   * call handleTouchEnd when touchend was called on selectedScreen
+   * @public
+   * @param {Event} event - event object
+   */
+  handleTouchEnd: function(event) {
+    event.preventDefault(); // to prevent mobile from propagating click to discovery shown on pause
```

Comment:
Actually it won't stop propagation, but stop default browser listeners to hear event exactly on touched element (not parent). I'm not aware yet about the reason of this method in general, but if you wanted to stop propagation you need event.stopPropagation() method. Is .preventDefault() is exactly what you wanted then probably we need another comment for this line.

Reply:
\+

Reply:
\+